googler

```diff
@@ -19,16 +19,21 @@
 
 from __future__ import print_function
 import argparse
+import atexit
 import collections
+import functools
```

Comment:
In decorators, if you don't wrap the decorated methods, you lose their signatures and docstrings. I'm not wrapping the `GoogleParser` methods because I consider it an opaque class (that's why the documentation there is in comments, not docstrings). But for `GooglerCmd`, I'm exposing the API, so the methods should be wrapped.